src/lib/api/api.go

```diff
@@ -558,27 +576,30 @@ func (a *API) GetLabels(w http.ResponseWriter, r *http.Request) {
 	}
 	a.ResponseStatus = http.StatusText(200)
 	a.Data = uniqueLabels
-	a.send(w, a.marshalled())
+	a.send(w, a.Data, nil)
 }
 
-func (a *API) marshalled() []byte {
+func marshalled(status string, data interface{}, message interface{}) []byte {
 	response := struct {
-		Status string      `json:"status"`
-		Data   interface{} `json:"data"`
+		Status  string      `json:"status"`
+		Data    interface{} `json:"data"`
+		Message interface{} `json:"message"`
 	}{
-		Status: a.ResponseStatus,
-		Data:   a.Data,
+		Status:  status,
+		Data:    data,
+		Message: message,
 	}
 	js, err := json.Marshal(response)
 	if err != nil {
-		panic(err)
+		fmt.Fprintln(os.Stderr, "json-marshal error: ", err)
```

Comment:
Why do we print the error at this point?

Reply:
Hello @aquibbaig, kindly check [this](https://github.com/bench-routes/bench-routes/pull/391#discussion_r598139953)

Reply:
It's fine to print than to panic :smiley:

Reply:
I'd say use "log.infoln"

Reply:
You mean log.Error?

Reply:
what changes should I make here @aquibbaig @Harkishen-Singh?

Reply:
```suggestion
		log.Error("json-marshal error: ", err.Error())
```

Reply:
Thank you!

Reply:
Yep, that looks fine!